evalml/objectives/binary_classification_objective.py

```diff
@@ -50,16 +50,14 @@ def optimize_threshold(self, ypred_proba, y_true, X=None):
 
         def cost(threshold):
             y_predicted = self.decision_function(
-                ypred_proba=ypred_proba, threshold=threshold, X=X
+                ypred_proba=ypred_proba, threshold=threshold[0], X=X
             )
             cost = self.objective_function(y_true, y_predicted, X=X)
             return -cost if self.greater_is_better else cost
 
-        optimal = minimize_scalar(
-            cost, bracket=(0, 1), method="Golden", options={"maxiter": 250}
-        )
+        optimal = differential_evolution(cost, bounds=[(0, 1)], seed=0)
```

Comment:
might be interesting to play with some of the parameters to see if we can get the same performance but quicker. Def not blocking though!

Reply:
From my impression, the default values should perform fairly similar in speed to the other optimization methods available to this. I can try running a few other perf tests to double check though!